src/calendar-heatmap.js

```diff
@@ -11,7 +11,7 @@ function calendarHeatmap() {
   var SQUARE_LENGTH = 13;
   var SQUARE_PADDING = 1;
   var MONTH_LABEL_PADDING = 6;
-  var now = moment().endOf('day').toDate();
+  var yesterday = moment().add(-1, 'days').endOf('day').toDate();
```

Comment:
Although correct, can you have this in the same order the yearAgo is written? Reads easier that way

`moment().endOf('day').subtract(1, 'day').toDate();`